packages/block-library/src/search/index.php

```diff
@@ -82,14 +82,15 @@ function render_block_core_search( $attributes ) {
 			$button_classes        .= ' has-icon';
 			$button_internal_markup =
 				'<svg id="search-icon" class="search-icon" viewBox="0 0 24 24" width="24" height="24">
-			        <path d="M13.5 6C10.5 6 8 8.5 8 11.5c0 1.1.3 2.1.9 3l-3.4 3 1 1.1 3.4-2.9c1 .9 2.2 1.4 3.6 1.4 3 0 5.5-2.5 5.5-5.5C19 8.5 16.5 6 13.5 6zm0 9.5c-2.2 0-4-1.8-4-4s1.8-4 4-4 4 1.8 4 4-1.8 4-4 4z"></path>
-			    </svg>';
+					<path d="M13.5 6C10.5 6 8 8.5 8 11.5c0 1.1.3 2.1.9 3l-3.4 3 1 1.1 3.4-2.9c1 .9 2.2 1.4 3.6 1.4 3 0 5.5-2.5 5.5-5.5C19 8.5 16.5 6 13.5 6zm0 9.5c-2.2 0-4-1.8-4-4s1.8-4 4-4 4 1.8 4 4-1.8 4-4 4z"></path>
+				</svg>';
 		}
 
 		$button_markup = sprintf(
-			'<button type="submit" class="wp-block-search__button %s" %s>%s</button>',
+			'<button type="submit" class="wp-block-search__button %s" %s aria-label="%s">%s</button>',
 			esc_attr( $button_classes ),
 			$inline_styles['button'],
+			esc_attr( $attributes['label'] ),
```

Comment:
It's possible that `label` could have HTML in it (see https://github.com/WordPress/gutenberg/pull/38649), which at least VoiceOver will read aloud - we can strip that out:

```suggestion
			esc_attr( strip_tags( $attributes['label'] ) ),
```

Reply:
Nice catch @ryelle ! I thought that `esc_attr` is already sanitizing and stripping tags. Will fix it with the next commit.

Reply:
Here is the docs for this: https://developer.wordpress.org/reference/functions/esc_attr/

It does not strip tags.

Anyway, thanks @Sisanu for fixing and @ryelle for catching. I did not know tags could be sent in via this attribute. This could have ended badly.